homeassistant/components/wallbox/strings.json

```diff
@@ -38,6 +38,9 @@
       },
       "energy_price": {
         "name": "Energy price"
+      },
+      "maximum_icp_current": {
+        "name": "Maximum ICP current"
```

Comment:
What's ICP

Reply:
I just learned that yesterday ;-) : "the function is "setIcpMaxCurrent" it is used to limit the power draw at the main meter for the whole house while "setMaxChargingCurrent" is used to limit the power output to the car. " So, you limit the power used by the wallbox by setting a upper limit for the entire house. Not sure what the abbreviation stands for though.

Reply:
if we don't know what the abbrevisation stands for, should we remove it?

Reply:
**Additional information:**

### Difference between `setIcpMaxCurrent` and `setMaxChargingCurrent`:

- **`setMaxChargingCurrent`** is a more static setting that determines the maximum charging power allowed by the Wallbox. For example, if my Wallbox's electrical line can support a maximum of 6 kW, I will set this value to 6 kW. Alternatively, if I do not want to charge my car above 5 kW for any reason, I can set it to 5 kW. This is a fixed limit and does not change dynamically based on real-time conditions.

- **`setIcpMaxCurrent`** is a dynamic limit. For instance, if `setIcpMaxCurrent` is set to 6 kW, but other appliances in my house are consuming 4 kW, the car's charging rate must be reduced to 2 kW to avoid exceeding my contractual power limit. If this limit is exceeded, my electrical line may be disconnected. Thus, the Wallbox, by knowing my contractual power limit through `setIcpMaxCurrent`, continuously adjusts the car's charging power dynamically, ensuring it does not exceed 6 kW, but adjusts downward as needed to prevent disconnection.

### Why it is important to add `setIcpMaxCurrent`:

Currently, the Wallbox integration in Home Assistant does not include a control for `setIcpMaxCurrent`. Adding this feature would allow users to dynamically manage their charging power in response to changing contractual power limits.

For example, during nighttime and holidays, my contractual power limit is 6 kW, while during the rest of the time, it is 3 kW. With `setIcpMaxCurrent` added to the integration, I could create automations in Home Assistant to automatically adjust this value to 6 kW during nights and holidays and to 3 kW during the rest of the time.

While the official Wallbox app (and the underlying library) already supports the `setIcpMaxCurrent` function, it does not have a scheduler. This limitation means that I would need to manually change this setting throughout the day, at night, and during holidays. By adding `setIcpMaxCurrent` to Home Assistant, users could automate these adjustments, significantly improving usability and flexibility.

Reply:
> What's ICP

It's the spanish abbreviation of "Interruptor de Control de Potencia", I guess the spanish term for the main breaker current limit